js/signin.js

```diff
@@ -0,0 +1,25 @@
+import {
+  checkemail,
+  checkpassword,
+  ablelogin,
+  changeeye,
+} from "./authFunctions.js";
+
+const email = document.querySelector(".input-email");
+const password = document.querySelector(".input-password");
+const loginbutton = document.querySelector("button");
+const emailerror = document.querySelector(".email-error");
+const passworderror = document.querySelector(".password-error");
+const eye = document.querySelector(".password-img1");
+
+email.addEventListener("focusout", function () {
+  checkemail(email, emailerror);
+  ablelogin(email, password, loginbutton);
+});
```

Comment:
지금 기준으로는, email을 검증하고, 로그인을 가능하게(enable)한다 정도로 보입니다.
즉, email 검증 , email검증후 메시지 표시, 로그인 활성화 표시 동작들을 하는 걸로 보이긴 하지만 조금더 드러나면 좋을거같네요. 이를 테면, 검증로직 / 결과에 따라, 로그인버튼 활성화 또는 에러메시지 표현이 이 함수안에서 드러나면 좋겠습니다. 조건문은 개별 함수 안보다는, 가급적 이렇게 상위로 올리는 편이 좋습니다. 대략적인 아이디어는 이렇습니다.

```suggestion
email.addEventListener("focusout", function () {
    const validation = validateEmail(/* 생략, 구현에 따라 다름 */);
    if(validation.result) {
      enableLogin(/* 생략, 구현에 따라 다름 */);
    } else {
      showErrorMessage(validation.errorMessage)
    }
});
```